package.json

```diff
@@ -8,7 +8,7 @@
   "author": "GitHub",
   "license": "MIT",
   "scripts": {
-    "start": "cross-env NODE_PATH=. NODE_ENV=production node server.js",
+    "start": "cross-env NODE_OPTIONS='--max_old_space_size=4096' NODE_PATH=. NODE_ENV=production node server.js",
```

Comment:
Is this because it's reading all of the blog post contents into memory?

Reply:
No, we just really load a big JSON file. It reproduced outside of this, in the bump PR it's also crashed.